chrome-launcher/chrome-launcher.ts

```diff
@@ -310,3 +367,9 @@ export class Launcher {
     });
   }
 };
+
+const manager = new LaunchManager();
```

Comment:
yeah good point, shouldn't be a problem to create one for each launch though should it?